plugins/rust-openid-token-plugin/README.md

```diff
@@ -139,6 +139,23 @@ and
 gpcurl ... | jq --raw-output .headers[0].value | base64 -d
 ```
 
+### Remove existing auth headers that share the same name
+
+You may be running with a collection or specification that already has an `Authorization: Bearer 123...` or some similar header. To dynamically remove those headers that are already present in the spec, you can add the `--overwrite-existing-headers` flag to the command line.
```

Comment:
I have the impression that since you are specifying this to be true by default, clap is showing `--no-overwrite-existing-headers` as the option.
Please check this out